utils/client.js

```diff
@@ -2,8 +2,10 @@ import ApolloClient from 'apollo-boost';
 import { InMemoryCache } from 'apollo-cache-inmemory';
 
 const PORT = process.env.PORT || 3000;
+const path = process.env.ENV_NODE ==='development'?`http://localhost:${PORT}`:'https://vcars.herokuapp.com';
```

Comment:
Codacy has a fix for the issue: [Operator '===' must be spaced.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7125582)
```suggestion
const path = process.env.ENV_NODE === 'development'?`http://localhost:${PORT}`:'https://vcars.herokuapp.com';
```